### DISCUSSION TITLE
some quesitons about HttpWasmAssetReader

### DISCUSSION THREAD

**Discussion by lizelive** [bevyengine/bevy#10568] [Ideas]
made [a http asset loader](https://github.com/lizelive/bevy_http)

what are the requirements for merging something like this into the engine?

**Comment by alice-i-cecile** [comment]
Generally just open a pull request and we can discuss it there! We'll want docs and tests :) In this particular case, it should also be behind a feature flag.

Useful functionality, but only to some users.

**Comment by mockersf** [comment]
This feels like something very good to keep as a third party plugin for now.

What would be the big advantage to integrate that in the engine?
